src/auth/index.ts

```diff
@@ -0,0 +1,88 @@
+/*!
+ * Copyright 2020 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as _actionCode from './action-code-settings-builder';
+import * as _auth from './auth';
+import * as _config from './auth-config';
+import * as _identifier from './identifier';
+import * as _import from './user-import-builder';
+import * as _mfa from './multi-factor';
+import * as _tenant from './tenant';
+import * as _tenantManager from './tenant-manager';
+import * as _user from './user-record';
+
+/* eslint-disable @typescript-eslint/no-namespace */
+export namespace admin.auth {
+    /* eslint-disable @typescript-eslint/no-empty-interface */
```

Comment:
The indentation seems to be off in this block

Reply:
Good catch. Fixed.